ndex2/client.py

```diff
@@ -950,6 +1327,26 @@ def update_network_user_permission(self, userid, networkid, permission):
     def grant_networks_to_group(self, groupid, networkids, permission="READ"):
         """
         Set group permission for a set of networks
+	
+	.. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		#find the group id by the username
+		user = "INSERT YOUR USERNAME HERE"
+		group_ids = [g.gr_gid for g in grp.getgrall() if user in g.gr_mem]
```

Comment:
The NDEx concept of groups is not the same as the ones exposed via the `pwd` module. Just set group_id = 'ENTER YOUR GROUP ID HERE' and omit this section

Reply:
Please replace the group_ids logic with just this line:
`group_ids = 'ENTER YOUR GROUP ID HERE'`